lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -334,12 +283,8 @@ sub asset_lookup {
 
 sub _decrease {
     my ($self, $size) = (shift, shift // 0);
-
-    my $log = $self->log;
-    $log->debug("Current cache size: $self->{cache_real_size}");
```

Comment:
did you move this message somewhere else? Because so far it proved useful for example in case when the "current size" showed up as 0 when it was clearly miscalculated, e.g. when the cache location was behind a symlink.